lib/elix/responders/gif_me.ex

```diff
@@ -0,0 +1,47 @@
+defmodule Elix.Responders.GifMe do
+  @moduledoc """
+  Searches the Giphy public API and return the URL
+  of a GIF matching the search term.
+  """
+
+  use Hedwig.Responder
+
+  @base_url "http://api.giphy.com/v1/gifs/search"
+  @api_key "dc6zaTOxFJmzC" # Giphy’s public beta API key
+  @sample_size 10
+
+  @usage """
+  gif me <term> - Replies with a GIF URL matching the term
+  """
+  hear ~r/gif me (.+)/i, msg do
```

Comment:
Nevermind, this won't match "want some gif meat", but it will match "he tried to punch.gif me in the face!" which, on second thought, might actually have comedic value...